metadata/utils.go

```diff
@@ -66,3 +69,17 @@ func TryGetPriority(props map[string]string) (uint8, bool, error) {
 
 	return 0, false, nil
 }
+
+// TryIsRawPayload determines if payload should be used as-is.
+func TryIsRawPayload(props map[string]string) (bool, error) {
```

Comment:
nit: Can you rename to `IsRawPayload` or even `RawPayload`? The `Try` is redundant here as.

Reply:
I used the Try prefix to be consistent with the other methods since it can fail to parse it and return error.